### PR TITLE
From browserstack to cbt

### PR DESCRIPTION
**Pull request by aril-spetalen**

### CODE REVIEW

**Comment by skjalgepalg** [comment]
remote tests on edge throws an error from selenium for edge on nested dialogs. feature has been verified manually.

File: readme.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ npm start # Your browser will open documentation with hot reloading
...
+Before running tests with CrossBrowserTesting you need a `.env` file providing credentials:
 
 ```sh
-BROWSERSTACK_USER=...
-BROWSERSTACK_KEY=...
+SMARTBEAR_USER=...
+SMARTBEAR_AUTHKEY=...
 ```
 
 Save it in the root directory. Now you can either run the tests locally:
```
Comment (eirikbacker) [inline readme.md:31]: Is the keys needed localy anymore?

Comment (skjalgepalg) [reply readme.md:31]: shouldn't be; will doublecheck and update readme accordingly

**Review by eirikbacker — APPROVED** [review]
Nice work! 💯

**Pull request merged**